### PR TITLE
DevTools 111: HD Colors

### PR DESCRIPTION
**Pull request by sofiayem**
Moved the color picker section into a separate doc because it gained sophistication.

LMK if you think we should remove the video from the doc altogether because it's now outdated.

Publish on March 7.

### CODE REVIEW

**Comment by sofiayem** [comment]
Ooops, forgot to:
- Deprecate the color option in Settings
- Document color-mix()

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/css/color/index.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+{% Aside %}
+By default, the **Color Picker** suggests you to follow the [WebAIM guidelines](https://webaim.org/standards/wcag/). Alternatively, you can set it to suggest values compliant with [APCA](https://web.dev/color-and-contrast-accessibility/#advanced-perceptual-contrast-algorithm-apca):
+
+In {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [**Settings** > **Experiments**](/docs/devtools/settings/experiments/), check {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hmp8j3HiLMCcqPArD9yt.svg", alt="Checkbox.", width="22", height="22" %} **Enable new Advanced Perceptual Contrast Algorithm (APCA)...**
+{% endAside %}
+
+To get a list of all contrast issues in one go, follow the [Make your website more readable](/docs/devtools/accessibility/contrast/) guide.
+
+## Sample a color anywhere with the Eyedropper {: #eyedropper }
```
Comment (jecfish) [inline site/en/docs/devtools/css/color/index.md:141]: There's a new keyboard shortcut for sampling color. See the WNDT 111.

**Review by jecfish — APPROVED** [review]

File: site/en/docs/devtools/css/color/index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+{% Aside "important" %}
+This video shows Chrome DevTools version earlier than 112. In later versions, the **Styles** pane supports:
+
+- 12 new color spaces and 7 new gamuts from the [CSS Color Level 4](https://www.w3.org/TR/css-color-4/) specification.
+- The [`color-mix()`](/blog/css-color-mix) function from [CSS Color Level 5](https://www.w3.org/TR/css-color-5/#color-mix).
+{% endAside %}
+
+For a deep dive on the new color spaces, see [High Definition CSS Color Guide](/articles/high-definition-css-color-guide/).
+
+## Open the Color Picker and change colors {: #change-colors }
```
Comment (jecfish) [inline site/en/docs/devtools/css/color/index.md:28]: Somewhere in this docs, mention that user can Shift + Click on the color preview icon to open the conversion dialog

File: site/en/docs/devtools/css/color/index.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VDuxFzZRnRkSWsSfTlhx.png", alt="The drop-down list with all the supported color spaces.", width="800", height="838" %}
+
+1. Choose one of the following color spaces:
+
+   - [`#<hex-color>`](https://developer.mozilla.org/docs/Web/CSS/hex-color)
+   - [`rgb(X X X / .X)`](https://developer.mozilla.org/docs/Web/CSS/color_value/rgb).
+   - [`hsl(X X X / X)`](https://developer.mozilla.org/docs/Web/CSS/color_value/hsl).
+   - [`hwb(X X X / .X)`](https://developer.mozilla.org/docs/Web/CSS/color_value/hwb).
+   
+   Or one of the experimental ones:
```
Comment (jecfish) [inline site/en/docs/devtools/css/color/index.md:96]: Are these color still experimental?

**Comment by jecfish** [comment]
LGTM overall, a few minor comments. Thanks Sofia!

File: site/en/docs/devtools/css/color/index.md (hunk 3 above)
Comment (sofiayem) [reply site/en/docs/devtools/css/color/index.md:96]: In the MDN docs that I reference, they are.

File: site/en/docs/devtools/css/color/index.md (hunk 2 above)
Comment (sofiayem) [reply site/en/docs/devtools/css/color/index.md:28]: I did in two instances.

File: site/en/docs/devtools/css/color/index.md (hunk 1 above)
Comment (sofiayem) [reply site/en/docs/devtools/css/color/index.md:141]: Done

File: site/en/docs/devtools/css/color/index.md (hunk 3 above)
Comment (sofiayem) [reply site/en/docs/devtools/css/color/index.md]: Replaced with new

**Pull request merged**